patches/colloid/fix-tooltip.patch

```diff
@@ -0,0 +1,90 @@
+diff --git a/src/sass/_colors.scss b/src/sass/_colors.scss
+index 8738bf37..432a0596 100644
+--- a/src/sass/_colors.scss
++++ b/src/sass/_colors.scss
+@@ -125,7 +125,7 @@ $surface:                               if($variant == 'light', background(a), b
+ $base:                                  if($variant == 'light', background(a), background(g));
+ $base-alt:                              if($variant == 'light', background(b), background(f));
+
+-$tooltip:                               if($variant == 'dark', rgba(darken(background(e), 3%), 0.9), rgba(darken(background(h), 3%), 0.9));
++$tooltip:                               $grey-100;
+ $osd:                                   background(e);
+ $scrim:                                 background(f);
+ $scrim-alt:                             background(g);
+diff --git a/src/sass/gtk/_common-3.0.scss b/src/sass/gtk/_common-3.0.scss
+index 0b6df5b8..ae7d0cbb 100644
+--- a/src/sass/gtk/_common-3.0.scss
++++ b/src/sass/gtk/_common-3.0.scss
+@@ -231,7 +231,7 @@ label {
+   &.osd {
+     border-radius: $corner-radius;
+     background-color: $tooltip;
+-    color: on($tooltip);
```

Comment:
Is there a reason we can't use this?

Reply:
the `on()` function generates a more appropriate color in this case black, so i chose to just change it.